### PR TITLE
dockerize 0.6.1 (new formula)

### PR DESCRIPTION
**Pull request by mrmachine**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/dockerize.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    (buildpath/"src/github.com/jwilder/dockerize").install buildpath.children
+    ENV.append_path "PATH", buildpath/"bin"
+    cd "src/github.com/jwilder/dockerize" do
+      system "make", "dist"
+      bin.install "dist/darwin/amd64/dockerize"
+    end
+  end
+
+  test do
+    system "dist/darwin/amd64/dockerize", "--help"
```
Comment (fxcoudert) [inline Formula/dockerize.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/dockerize.rb (hunk 1 above)
Comment (ruudk) [reply Formula/dockerize.rb]: @mrmachine Can you add some tests? This recipe is really nice to have in homebrew

Comment (ruudk) [reply Formula/dockerize.rb]: What about 
```
dockerize -wait http://www.google.com -wait-retry-interval=1s -timeout 5s
```

File: Formula/dockerize.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  url "https://github.com/jwilder/dockerize.git",
+      :tag      => "v0.6.1",
+      :revision => "7c5cd7c34dcf1c81f6b4db132ebceabdaae17153"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/jwilder/dockerize").install buildpath.children
+    ENV.append_path "PATH", buildpath/"bin"
```
Comment (igas) [inline Formula/dockerize.rb]: I don't think this is needed

File: Formula/dockerize.rb (hunk 1 above)
Comment (igas) [reply Formula/dockerize.rb]: You don't have access to build time files, this should be `bin/"dockerize"`

**Comment by ruudk** [comment]
@mrmachine Any news on this? Would be great to be able to install this with Brew.

**Comment by ruudk** [comment]
@igas What is needed to get this merged? Maybe you can explain how I can fix this PR? I really would like to see this merged.

**Comment by fxcoudert** [comment]
@ruudk the test is this issue, right on. See review comments above.

File: Formula/dockerize.rb (hunk 1 above)
Comment (ruudk) [reply Formula/dockerize.rb]: As @mrmachine doesn't respond anymore, I created a new PR to address this: https://github.com/Homebrew/homebrew-core/pull/37407.

Please close this PR.

File: Formula/dockerize.rb (hunk 2 above)
Comment (ruudk) [reply Formula/dockerize.rb]: That's needed for the Makefile. It installs `glock` via `go get` and then tries to execute it.

As @mrmachine doesn't respond anymore, I created a new PR to address this: https://github.com/Homebrew/homebrew-core/pull/37407.

Please close this PR.

**Comment by fxcoudert** [comment]
Merged in #37407

**Pull request closed without merging**